docs/_includes/Types.md

```diff
@@ -2905,25 +2902,25 @@ where the type of `mid` is inferred to be `int`.  Since the result
 value of the division is a member of type `int32`, one can introduce
 yet another conversion operation to make the type of `mid` be `int32`:
 ```dafny
-var mid := int32(lo as int + hi as int) / 2);
+var mid := ((lo as int + hi as int) / 2) as int32;
 ```
 If the compiler does specialize the run-time representation for
 `int32`, then these statements come at the expense of two,
 respectively three, run-time conversions.
 
 The `as N` conversion operation is grammatically a suffix operation like
-`.`_field and array indsexing. Thus the as operawtion binds more tightly than
-p[refix or binary operations: `- x as int` is `- (x as int)`; `a + b as int` is `a + (b as int)`.
-
-TODO: Move this section to Expressions?
+`.`_field and array indsexing. Thus the `as` operation binds more tightly than
```

Comment:
```suggestion
`.`_field and array indexing. Thus the `as` operation binds more tightly than
```

(I'm not sure if the `_` is a typo or an intentional type-setting command.)

Reply:
Fixed